src/Avatar/BehaviorPipeline.cs

```diff
@@ -110,9 +110,9 @@ public IMethodReturn Invoke(IMethodInvocation invocation, bool throwOnException
                 return (index < behaviors.Length) ?
                     behaviors[index].Execute :
                     (m, n) => CallBaseOrThrow(m);
-            };
+            }
 
-            var result = behaviors[index].Execute(invocation, (m, n) => getNext().Invoke(m, n));
+            var result = behaviors[index].Execute(invocation, (m, n) => GetNext().Invoke(m, n));
```

Comment:
I haven't tried this myself but... while we're at it, could `(m, n) => GetNext().Invoke(m, n)` possibly be simplified to just `GetNext()`?

Reply:
Sorry, please ignore. `GetNext` obviously needs to be invoked repeatedly, so this simplification wouldn't work.